lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -35,62 +35,29 @@ def setUp(self):
         }
         json_analytics_data = json.dumps(analytics_data)
         self.data = {"data": json_analytics_data}
+        self.zendesk_response = ('A problem has occurred retrieving the data, to report the problem click '
```

Comment:
Should the URL be parameterized to use the same value passed to `override_settings`?